fv3gfs/physics/testing/translate_physics.py

```diff
@@ -0,0 +1,69 @@
+from fv3core.testing import TranslateFortranData2Py
+import numpy as np
+import fv3core.utils.gt4py_utils as utils
+
+
+def read_serialized_data(serializer, savepoint, variable):
+    data = serializer.read(variable, savepoint)
+    n_dim = len(data.shape)
+    cn = int(np.sqrt(data.shape[0]))
+    npz = data.shape[1]
```

Comment:
some serialized data is scalars (though is read by serialbox as an array of lenght 1), and this line would crash.

Reply:
Good catch, adding type check before shape calls